src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
```

Comment:
```suggestion
    public Ball getRandomBall() {
```

логичней что б наш метод getRandomBall возвращал именно экземпляр класса Ball